platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/event/preconsensus/AsyncPreconsensusEventWriterTests.java

```diff
@@ -246,6 +240,10 @@ private PlatformContext buildContext() {
         final Configuration configuration = new TestConfigBuilder()
                 .withValue("event.preconsensus.databaseDirectory", testDirectory)
                 .withValue("event.preconsensus.preferredFileSizeMegabytes", 5)
+                .withValue("transaction.maxTransactionBytesPerEvent", Integer.MAX_VALUE)
+                .withValue("transaction.maxTransactionCountPerEvent", Integer.MAX_VALUE)
+                .withValue("transaction.transactionMaxBytes", 245760)
```

Comment:
Because of the `@Max` annotation on the Config - This annotation came from the Implementation in `Settings.java`. I've misinterpreted ` if (maxTransactionBytesPerEvent < transactionMaxBytes)` wrong. 

Fixed it.